Source/Tools/FEXBash/FEXBash.cpp

```diff
@@ -25,11 +24,21 @@ int main(int argc, char** argv, char** const envp) {
   // Use /bin/sh for -c commands and /bin/bash for interactive mode
   const char* BashPath = Args.empty() ? "/bin/bash" : "/bin/sh";
 
-  std::string FEXInterpreterPath = std::filesystem::path(argv[0]).parent_path().string() + "FEXInterpreter";
+  std::string FEXInterpreterPath = std::filesystem::path(argv[0]).parent_path().string() + "/FEXInterpreter";
```

Comment:
This seemingly was broken before, since `parent_path` does not include a trailing `/`. Instead, paths like `/usr/binFEXInterpreter` where checked.